bundle/Form/Type/RelatedContentFilterType.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+namespace Netgen\TagsBundle\Form\Type;
+
+use eZ\Publish\API\Repository\ContentTypeService;
+use eZ\Publish\API\Repository\Exceptions\NotFoundException;
+use eZ\Publish\API\Repository\Values\Content\Query\FacetBuilder\ContentTypeFacetBuilder;
+use eZ\Publish\API\Repository\Values\Content\Search\Facet\ContentTypeFacet;
+use Netgen\TagsBundle\API\Repository\TagsService;
+use Netgen\TagsBundle\API\Repository\Values\Tags\Tag;
+use Netgen\TagsBundle\Core\Search\RelatedContent\SortService;
+use Netgen\TagsBundle\Exception\FacetingNotSupportedException;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class RelatedContentFilterType extends AbstractType
+{
+    /**
+     * @var \Netgen\TagsBundle\API\Repository\TagsService
+     */
+    protected $tagsService;
+
+    /**
+     * @var \eZ\Publish\API\Repository\ContentTypeService
+     */
+    protected $contentTypeService;
+
+    /**
+     * @var \Netgen\TagsBundle\Core\Search\RelatedContent\SortService
+     */
+    protected $sortService;
+
+    /**
+     * ContentTypeFilterType constructor.
+     *
+     * @param \Netgen\TagsBundle\API\Repository\TagsService $tagsService
+     * @param \eZ\Publish\API\Repository\ContentTypeService $contentTypeService
+     * @param \Netgen\TagsBundle\Core\Search\RelatedContent\SortService $sortService
+     */
+    public function __construct(TagsService $tagsService, ContentTypeService $contentTypeService, SortService $sortService)
+    {
+        $this->tagsService = $tagsService;
+        $this->contentTypeService = $contentTypeService;
+        $this->sortService = $sortService;
+    }
+
+    /**
+     * {@inheritdoc}
```

Comment:
No need for `@inheritdoc` here. PHPDocs are inherited by default.

Reply:
Fixed in https://github.com/netgen/TagsBundle/pull/98/commits/05f7dad27c86fbfdc247c7f08f2966bc63b3a183.